packages/garbo/src/fights.ts

```diff
@@ -528,29 +557,15 @@ export function dailyFights(): void {
 
         const location = new EmbezzlerFightRunOptions(nextFight).location;
         const underwater = location.environment === "underwater";
-        const shouldCopy = get("_badlyRomanticArrows") === 0 && !underwater;
 
-        // use obtuse angel if have + have quake of arrows, otherwise reanimator
-        // quake of arrows is PvP-stealable and costs ~50k, so don't assume we have it
-        let bestCopier: Familiar | undefined;
-        if (
-          have($familiar`Obtuse Angel`) &&
-          (familiarEquippedEquipment($familiar`Obtuse Angel`) ===
-            $item`quake of arrows` ||
-            retrieveItem($item`quake of arrows`))
-        ) {
-          bestCopier = $familiar`Obtuse Angel`;
-        } else if (have($familiar`Reanimated Reanimator`)) {
-          bestCopier = $familiar`Reanimated Reanimator`;
-        }
-        const familiar = shouldCopy && bestCopier ? bestCopier : meatFamiliar();
-        const famSpec: OutfitSpec = { familiar };
-        if (familiar === $familiar`Obtuse Angel`) {
-          famSpec.famequip = $item`quake of arrows`;
+        const famSpec = () => familiarSpec(underwater);
```

Comment:
I don't know why you're making this a function

Reply:
Paranoia.